cart.spec

```diff
@@ -0,0 +1,82 @@
+Name:          cart
+Version:       0.0.1
+Release:       1%{?dist}
+Summary:       CaRT
+
+License:       Apache
+URL:           https//github.com/daos-stack/cart
+Source0:       %{name}-%{version}.tar.gz
+Source1:       scons_local-%{version}.tar.gz
+
+BuildRequires: scons
+BuildRequires: libfabric-devel
+BuildRequires: pmix-devel
+BuildRequires: openpa-devel
+BuildRequires: mercury-devel
+BuildRequires: ompi-devel
+BuildRequires: libevent-devel
+BuildRequires: boost-devel
+BuildRequires: libuuid-devel
+BuildRequires: hwloc-devel
+BuildRequires: openssl-devel
+BuildRequires: libcmocka-devel
+BuildRequires: libyaml-devel
+#Requires:
+
+%description
+
+Collective and RPC Transport (CaRT)
+
+CaRT is an open-source RPC transport layer for Big Data and Exascale
+HPC. It supports both traditional P2P RPC delivering and collective RPC
+which invokes the RPC at a group of target servers with a scalable
+tree-based message propagating.
+
+%prep
+%setup -q
+%setup -q -a 1
+
+
+%build
+# remove rpathing from the build
+find . -name SConscript | xargs sed -i -e '/AppendUnique(RPATH=.*)/d'
+
+scons %{?_smp_mflags}                     \
+      --config=force                      \
+      --update-prereq=all                 \
+      --build-deps=yes                    \
```

Comment:
What deps is this actually building, this specfile looks to have listed them all as dependencies already.  I think it would be possible to put a comma separated list here if there are a couple that we want built, and that would be much more failsafe against future changes.

Reply:
Again, just one of the "standard" options we have successfully used.

Reply:
The risk here is the list of dependances is wrong, or changes somehow and the build ends up unintentionally changing so the CaRT rpm starts also including some of the dependances automatically, and we don't notice.  It would be nice to see what happens without this, and as I say to whitelist ones we actually do need, rather than have scons just fix any omissions

Reply:
Right.  I have a patch to remove `--update-prereq=all` and `--build-deps=yes` that I'm getting ready to push.